devicon.json

```diff
@@ -7614,6 +7614,24 @@
         "color": "#41B883",
         "aliases": []
     },
+   {
+        "name": "v8",
+	"altnames": [],
+        "tags": [
+            "javascript-runtime"
```

Comment:
```suggestion
            "javascript-runtime",
            "framework",
            "javascript",
            "web",
            "cpp"
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Tags:** Not secure